src/WebJobs.Script/Rpc/LanguageWorkerChannelException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    internal class LanguageWorkerChannelException : Exception
+    {
+        internal LanguageWorkerChannelException(LanguageWorkerState languageWorkerChannelState, string message) : base(message, new AggregateException(languageWorkerChannelState.Errors.ToList()))
```

Comment:
This will be problematic if `languageWorkerChannelState` or `Errors` are ever `null`.